compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs

```diff
@@ -67,11 +67,11 @@ featureAnyType = Feature
    , featureCppFlag = "DAML_ANY_TYPE"
    }
 
-featureTemplateTypeRep :: Feature
-featureTemplateTypeRep = Feature
-    { featureName = "templateTypeRep builtin"
+featureTypeRep :: Feature
```

Comment:
@cocreature should `featureTypeRep` be in the `allFeatures` list below?

Reply:
Good catch, fixed!